File: myqqrobot/myqqrobot/src/main/java/feijiang/robot/basic/message/MyListener.java
package feijiang.robot.basic.message;

import love.forte.di.annotation.Beans;
import love.forte.simboot.annotation.Listener;
import love.forte.simbot.event.FriendMessageEvent;

@Beans
public class MyListener {

    @Listener
    public void listen(FriendMessageEvent event) {
        // do...
        System.out.println(event.getId()+"发来了消息");
    }
}

File: myItem/myqqrobot/src/main/kotlin/feijiang/robot/core/listener/MyListener.kt
package feijiang.robot.core.listener

import love.forte.di.annotation.Beans
import love.forte.simboot.annotation.Listener
import love.forte.simbot.event.FriendMessageEvent

@Beans
class MyListener {

    @Listener
    suspend fun FriendMessageEvent.onEvent() {
        // do..
        println("我是小红")
    }
}